src/cmd/cmd.cpp

```diff
@@ -442,6 +442,11 @@ int main(int argc, char **argv)
     loop.exec();
 #endif
```

Comment:
Looks good.  Just a detail: you need to put the code before the #endif  even if it does not really matter since Qt 4 is no longer supported.  (But then we should remove the #if